.github/ISSUE_TEMPLATE/docs-feedback.yml

```diff
@@ -0,0 +1,47 @@
+name: Learn feedback control.
+description: |
+  ⛔ This template is hooked into the feedback control on the bottom of every page on the learn.microsoft.com site. It automatically fills in several fields for you. Don't use for other purposes. ⛔
+assignees:
+  - BillWagner
+body:
```

Comment:
Can we add a `labels` section and apply some label like `speclet issue` or something like that?

Reply:
We'll need to make the label as well.

Reply:
The labels are added as a config in the docs repo. What we found during initial testing is that when the URL targets a specific issue template, some of the items in the template aren't honored (title and labels). So, we add add those in the docfx.json file in the docs repo, and the labels get passed on the URL.

I was thinking of adding the "not triaged" label, and a new "area-speclet" label.

Reply:
> The labels are added as a config in the docs repo.

The problem, though, is that this template will show up when people create a new issue from this repo as well. If they do, the right labels won't be applied.

Reply:
Good point. I'll add them in both.